float32 setpoint
float64 time
string status